lib/ddtrace/contrib/redis/patcher.rb

```diff
@@ -30,8 +30,8 @@ def patch
 
               patch_redis()
               patch_redis_client()
-
               @patched = true
+              RailsCachePatcher.reload_cache_store if Rails::Patcher.patched?
```

Comment:
I don't like to have this here, but I can't see another way to approach this (`redis` can be patched before `rails` or vice-versa). Anyway, this line should be innocuous in environments without rails, because the `Rails::Patcher` is always "available" and the constant lookup for `RailsCachePatcher` only happens if `patched` is true.